brew_view/metrics.py

```diff
@@ -0,0 +1,109 @@
+import datetime
+from prometheus_client import Gauge, Counter, Summary
+
+from bg_utils.models import Request
+from brewtils.models import Request as BrewtilsRequest
+
+# Summaries:
+http_api_latency_total = Summary(
+    'bg_http_api_latency_seconds',
+    'Total number of seconds each API endpoint is taking to respond.',
+    ['method', 'route', 'status']
+)
+plugin_command_latency = Summary(
+    'bg_plugin_command_latency_seconds',
+    'Total time taken for a command to complete in seconds.',
+    ['system', 'instance_name', 'system_version', 'command', 'status'],
+)
+
+# Counters:
+completed_request_counter = Counter(
+    'bg_completed_requests_total',
+    'Number of completed requests.',
+    ['system', 'instance_name', 'system_version', 'command', 'status'],
+)
+request_counter_total = Counter(
+    'bg_requests_total',
+    'Number of requests.',
+    ['system', 'instance_name', 'system_version', 'command'],
+)
+
+# Gauges:
+queued_request_gauge = Gauge(
+    'bg_queued_requests',
+    'Number of requests waiting to be processed.',
+    ['system', 'instance_name', 'system_version'],
+)
+in_progress_request_gauge = Gauge(
+    'bg_in_progress_requests',
+    'Number of requests IN_PROGRESS',
+    ['system', 'instance_name', 'system_version'],
+)
+
+
+def initialize_counts():
+    for request in Request.objects(status='CREATED'):
```

Comment:
Just a thought really, but since we're querying the same collection, should we minimize network calls and just do `Request.objects(status__in=['CREATED', 'IN_PROGRESS'])` and then modify the for loop. Since this is happening at start-up, I think it matters a little bit less, but no need to make round trips if we don't have to is my thought.

Reply:
:+1: